/follow_me/laser_to_img:
  ros__parameters:
    # 縮小サイズを取得. 1[px] = 0.01[m]
    discrete_size: 0.01
    # Max LiDAR Range
    max_lidar_range: 3.5
    # 画像を表示するフラッグ
    img_show_flg: False

/follow_me/person_detector:
  ros__parameters:
    # 追従対象者との距離
    target_dist: 0.5
    # 起動時に追従対象者を検出するまでの待機時間
    init_time: 3.0
    # 起動時に追従対象を検出するまでのflg
    none_person_flg: True
    # 追従対象を見失ったときに追従を再開する時の距離の誤差
    target_diff: 0.5
  
/follow_me/base_controller:
  ros__parameters:
    # ロボットからみてtolerance[°]以内だったら積分制御しない視野角
    tolerance: 1.0
    # 積分制御をし始める視野角[°]
    i_range: 3.0
    # 目標座標周りの範囲
    target_width: 0.1
    target_height: 0.1
    # 並進のPゲイン==========================
    lkp: 0.3
    # 旋回のPIDゲイン========================
    # Pゲイン
    akp: 0.005
    # Iゲイン
    aki: 0.0
    # Dゲイン
    akd: 0.0
